cypress/integration/masterdata/bpartner_group_spec.js

```diff
@@ -36,14 +36,14 @@ describe('Create new bpartner group, https://github.com/metasfresh/metasfresh-e2
   //create bpartner
   it('Create Testpartner', function() {
     cy.visitWindow('123', bpartnerID);
-    cy.selectInListField('C_BP_Group_ID', GroupName, false /*modal*/, '/rest/api/window/.*' /*rewriteUrl*/);
+    cy.selectInListField('C_BP_Group_ID', groupName, false /*modal*/, '/rest/api/window/.*' /*rewriteUrl*/);
```

Comment:
I think here we can remove '/rest/api/window/' and it should still work since cy.selectInListField expects by default a patch from RewriteURL.Generic.